libvast/test/taxonomies.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright Tenzir GmbH. All rights reserved.
+
+#define SUITE taxonomies
+
+#include "vast/taxonomies.hpp"
+
+#include "vast/test/test.hpp"
+
+#include "vast/concept/parseable/to.hpp"
+#include "vast/concept/parseable/vast/expression.hpp"
+#include "vast/expression.hpp"
+
+#include <caf/test/dsl.hpp>
+
+using namespace vast;
+
+TEST(concepts - simple) {
+  auto c = concepts_type{{"foo", {"a.fo0", "b.foO", "x.foe"}},
+                         {"bar", {"a.bar", "b.baR"}}};
+  auto t = taxonomies{std::move(c), models_type{}};
+  {
+    MESSAGE("LHS");
+    auto exp = unbox(to<expression>("foo == \"1\""));
```

Comment:
Why do you use a string on the RHS? It has to be escaped and makes it harder on the eyes than necessary.